test/common/config/utility_test.cc

```diff
@@ -74,9 +74,6 @@ TEST(UtilityTest, TranslateApiConfigSource) {
   Utility::translateApiConfigSource("test_rest_legacy_cluster", 10000,
                                     ApiType::get().UnsupportedRestLegacy,
                                     api_config_source_rest_legacy);
-  EXPECT_EQ(
```

Comment:
Can you update this test to use some supported API type?

Reply:
Done. Sorry for delay :(